book/src/kimchi/foreign_field_add.md

```diff
@@ -120,7 +120,7 @@ bits  0..............87|88...........175|176...........263
 
 r  =  (-------r0-------|-------r1-------|-------r2-------)
 +
-g  =  (-------g0-------|-------g1-------|-------g2-------)
+b  =  (-------b0-------|-------b1-------|-------b2-------)
```

Comment:
```suggestion
g  =  (-------g0-------|-------g1-------|-------g2-------)
```

Reply:
We are intentionally using `g` to name $2^{264}$, so I suggest we don't use `b` here as it is misleading. Instead, maybe let's add a clarification of what `g` is meant to be.